targets/TARGET_Silicon_Labs/TARGET_EFM32/emlib/src/em_system.c

```diff
@@ -62,24 +61,19 @@ void SYSTEM_ChipRevisionGet(SYSTEM_ChipRevision_TypeDef *rev)
 
   EFM_ASSERT(rev);
 
-  uint32_t pid0 = SECURE_READ(&(ROMTABLE->PID0));
-  uint32_t pid1 = SECURE_READ(&(ROMTABLE->PID1));
-  uint32_t pid2 = SECURE_READ(&(ROMTABLE->PID2));
-  uint32_t pid3 = SECURE_READ(&(ROMTABLE->PID3));
```

Comment:
uVisor won't work if `SECURE_READ` is no longer used. This is why uVisor CI is failing. Please ensure that the `SECURE` macros continue to be used across updates to the Gecko SDK.